src/main/java/de/embl/cba/mobie2/display/SegmentationDisplay.java

```diff
@@ -75,7 +78,50 @@ public boolean showScatterPlot()
 		return tables;
 	}
 
-	public void updateSerializableFields(){
-		// TODO
-	};
+	/**
+	 * Create a serializable copy
+	 *
+	 * @param segmentationDisplay
+	 */
+	public SegmentationDisplay( SegmentationDisplay segmentationDisplay )
+	{
+		this.name = segmentationDisplay.name;
+		this.sources = new ArrayList<>();
+		for ( SourceAndConverter< ? > sourceAndConverter : segmentationDisplay.sourceAndConverters )
+		{
+			sources.add( sourceAndConverter.getSpimSource().getName() );
+		}
+
+		final SourceAndConverter< ? > sourceAndConverter = segmentationDisplay.sourceAndConverters.get( 0 );
+
+		if( sourceAndConverter.getConverter() instanceof AdjustableOpacityColorConverter)
+		{
+			this.opacity = ( ( AdjustableOpacityColorConverter ) sourceAndConverter.getConverter() ).getOpacity();
+		}
+
+		this.lut = segmentationDisplay.tableViewer.getColoringLUTName();
```

Comment:
@K-Meech I would prefer to fetch this information from `segmentationDisplay.coloringModel`? Can we get it from there? I think so... if not yet, we can add getter methods to MoBIEColoringModel.

Reply:
I moved the lut getter from the tableViewer to the coloringModel - so this should all work now.